modules/reissue-internal-certificates-secured-clusters.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+:_module-type: PROCEDURE
+[id="reissue-internal-certificates-secured-cluster_{context}"]
+= Reissuing internal certificates for Secured Clusters using init bundles
+
+Secured clusters contain the Collector, Sensor, and Admission Control components. These components use a built-in server certificate for authentication when communicating with other {product-title} components.
+
+The RHACS portal shows an information banner when the Central certificate is about to expire.
+
+[NOTE]
+====
+The information banner only appears 15 days before the certificate expiry date.
+====
+
+.Prerequisites
+
+* To reissue certificates, you must have `write` permission for the `ServiceIdentity` resource.
+
+[IMPORTANT]
+====
+Store this bundle securely because it contains secrets.
+You can use the same bundle on multiple secured clusters.
+====
+
+.Procedure
+
+To generate an init bundle using the {product-title-short} portal:
+
+. Select *Platform Configuration* -> *Clusters*.
+. Click *Manage Tokens*.
+. Navigate to the *Authentication Tokens* section, and click *Cluster Init Bundle*.
+. Click *Generate bundle*.
+. Enter a name for the cluster init bundle and click *Generate*.
+. Click *Download Kubernetes secrets file* to download the generated bundle.
+
+To generate an init bundle using the `roxctl` CLI, run the following command:
+
+[source,terminal]
+----
+$ roxctl -e <endpoint> -p <admin_password> central init-bundle generate <bundle_name> --output-secrets init-bundle.yaml
+----
+
+.Next steps
+
+* To create the necessary resources on each secured cluster, run the following command:
+
+[source,terminal]
+----
+$ oc -n stackrox apply -f <init-bundle.yaml>
+----
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/install-ocp-operator.adoc#create-resource-init-bundle[Creating resources by using the init bundle]
```

Comment:
These lines need to be removed from the module and added to the assembly right under the `include` for this module.